recipes/ecfeed/all/conanfile.py

```diff
@@ -0,0 +1,34 @@
+from conans import ConanFile, tools
+
+required_conan_version = ">=1.33.0"
+
+class EcFeedConan(ConanFile):
+    name = "ecfeed"
+    version = "1.1.0"
+    license = "EPL-1.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://ecfeed.com/"
+    description = "An open library used to connect to the ecFeed service. \
+        It can be integrated with the most common testing frameworks \
+        and generates a stream of test cases using a selected algorithm \
+        (e.g. Cartesian, N-Wise)."
+    topics = ("test", "ecfeed", "nwise")
+    build_policy = 'missing'
+    requires = "libcurl/7.72.0", "openssl/1.1.1c"
+    no_copy_source = True
```

Comment:
```suggestion
    no_copy_source = True
    settings = "os", "build_type", "arch", "compiler"
```

Keep full settings declared by default. The package id is manipulated by self.info.header_only()

Reply:
Updated.